### PR TITLE
Doc: Jormungandr configuration, running and troubleshooting

### PR DESCRIPTION
**Pull request by nonifier**

### CODE REVIEW

**Comment by nonifier** [comment]
Anyone knows how to create ***relative*** links to files using the markdown by any chance ?!? 
Just cannot get it to work.....

File: source/jormungandr/readme.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+This helps you overwrite the default settings if needed (from [default_settings.py](https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/default_settings.py)).
+
+# Run 
+
+To run the web service, you'll need to:
+
+* set `JORMUNGANDR_INSTANCES_DIR` to point at the directory that contains your `jormungandr.json`.
+* add `navitia/source/jormungandr` and `navitia/navitiacommon` to your python path via `PYTHONPATH`.
+* [optionaly] set `JORMUNGANDR_CONFIG_FILE` to point at your `custom_settings.py`.
```
Comment (kinnou02) [inline source/jormungandr/readme.md:73]: Mixing custom `settings.py` and configuration by environnement variable might not be the best idea, I would recommend using one or the other.
Typically I only use env var for my local jormungandr.

**Review by pbougue — COMMENTED** [review]
:tada: I agree with the mini comment from @kinnou02 , but great job on documenting this!

File: source/jormungandr/readme.md (hunk 1 above)
Comment (nonifier) [reply source/jormungandr/readme.md:73]: I've updated the doc mentioning that we prefer envvars over custom settings (ie. 
e9d7de9)

**Review by pbougue — APPROVED** [review]

File: source/jormungandr/readme.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+```json
+{
+    "key": "some_region",
+    "socket": "ipc:///tmp/default_kraken"
+}
+```
+
+You can copy the one from [documentation/examples/config/Jormungandr.json](https://github.com/CanalTP/navitia/blob/dev/documentation/examples/config/Jormungandr.json)
+
+Make sure `key` and `socket` respectively match `instance_name` and `zmp_socket` from your Kraken config file ([kraken.ini](https://github.com/CanalTP/navitia/blob/dev/documentation/examples/config/kraken.ini)).
```
Comment (TeXitoi) [inline source/jormungandr/readme.md]: `zmq_socket` ?

File: source/jormungandr/readme.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+## Environement Variables
+
+Environment variables can be set when the service is started. Referer to [default_settings.py](https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/default_settings.py)
+
+example for a devlopment environment: 
+
+```sh
+PYTHONPATH=..:../../navitiacommon/ JORMUNGANDR_INSTANCES_DIR=~/jormung_conf/ JORMUNGANDR_START_MONITORING_THREAD=False  JORMUNGANDR_DISABLE_DATABASE=True JORMUNGANDR_IS_PUBLIC=True python manage.py runserver
```
Comment (TeXitoi) [inline source/jormungandr/readme.md:54]: Maybe talk about json in instance env var?

**Review by TeXitoi — CHANGES_REQUESTED** [review]

File: source/jormungandr/readme.md (hunk 3 above)
Comment (nonifier) [reply source/jormungandr/readme.md:54]: now added in a685f32

**Review by kinnou02 — APPROVED** [review]

**Review by TeXitoi — APPROVED** [review]

**Review by mbounabi — APPROVED** [review]

File: source/jormungandr/readme.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+Make sure `key` and `socket` respectively match `instance_name` and `zmq_socket` from your Kraken config file ([kraken.ini](https://github.com/CanalTP/navitia/blob/dev/documentation/examples/config/kraken.ini)).
+
+As an alternative, you can also give the json through an environement variable like:
+```sh
+JORMUNGANDR_INSTANCE_FR_IDF='{"key": "fr-idf","zmq_socket": "tcp://localhost:3000"}'
+```
+
+
+## Environement Variables
```
Comment (pbougue) [inline source/jormungandr/readme.md]: typo: environment (no `e`)

**Pull request merged**